Source/interfaces/IDisplayInfo.h

```diff
@@ -70,6 +112,9 @@ namespace Exchange {
         virtual uint32_t VerticalFreq() const = 0;
         virtual HDRType Type() const = 0;
         virtual HDCPProtectionType HDCPProtection() const = 0;
+        virtual void HDCPProtection(const HDCPProtectionType) = 0;
+        virtual IHdrIterator* GetTvHdrCapabilities() const = 0;
+        virtual IHdrIterator* GetStbHdrCapabilities() const = 0;
```

Comment:
@pwielders, Based on your comment https://github.com/rdkcentral/Thunder/pull/214#discussion_r467864644, I have moved the HDR & HDMI related methods to Connection Properties interface, Could you please confirm if these changes are in alignment with your expectations?